{"name": "Проверяем, что нельзя изменить имя пользователя без авторизации", "status": "passed", "start": 1731119424858, "stop": 1731119425025, "uuid": "8b72c776-9011-45b0-9bfc-1c080b601632", "historyId": "0abe8efcbb9d0b00e1224b6e82d37a99", "testCaseId": "0abe8efcbb9d0b00e1224b6e82d37a99", "fullName": "tests.test_change_user_data.TestChangeUserData#test_change_name_unauthorized_returns_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "DESKTOP-0KFG2SR"}, {"name": "thread", "value": "12036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}